### PR TITLE
Add a new build configuration for a Beta version of LottieViewer.

### PR DESCRIPTION
**Pull request by simeoncran**
The "BETA" build configuration will build LottieViewer with a different name and identity so that it can be released to beta testers and exist side-by-side with the non-beta version.

### CODE REVIEW

**Comment by derobinsMSFT** [comment]
> CodeFlow for GitHub does not download executable files. If you trust this file and would like to run it, please download it yourself through GitHub.

Where does this file get used? Should there just be a PNG for this image?

---
Refers to: LottieViewer/Assets/Square150x150Logo-BETA.scale-200.pdn:1 in 75abbf5. [](commit_id = 75abbf53ded422e13e32c19d5c3b3f2d28ea205f, deletion_comment = False)

**Review by derobinsMSFT — APPROVED** [review]
:shipit:

**Comment by simeoncran** [comment]
> CodeFlow for GitHub does not download executable files. If you trust this file and would like to run it, please download it yourself through GitHub.

Oops... that's from Paint .NET that I used to modify all these files. Shouldn't be here.

---
In reply to: [724263298](https://github.com/windows-toolkit/Lottie-Windows/pull/380#issuecomment-724263298) [](ancestors = 724263298)

---
Refers to: LottieViewer/Assets/Square150x150Logo-BETA.scale-200.pdn:1 in 75abbf5. [](commit_id = 75abbf53ded422e13e32c19d5c3b3f2d28ea205f, deletion_comment = False)

File: LottieViewer/LottieViewer.csproj [hunk 1]
```diff
@@ -134,6 +134,70 @@
...
+    <Optimize>true</Optimize>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <NoWarn>1591;2008;8305</NoWarn>
+    <NoStdLib>true</NoStdLib>
+    <DebugType>pdbonly</DebugType>
+    <PlatformTarget>x86</PlatformTarget>
+    <UseVSHostingProcess>false</UseVSHostingProcess>
+    <LangVersion>latest</LangVersion>
+    <ErrorReport>prompt</ErrorReport>
+    <CodeAnalysisRuleSet>F:\github3\Lottie-Windows\Toolkit.ruleset</CodeAnalysisRuleSet>
```
Comment (derobinsMSFT) [inline LottieViewer/LottieViewer.csproj:150]: >F: [](start = 25, length = 2)

Can these rule set paths be made relative?

Comment (simeoncran) [reply LottieViewer/LottieViewer.csproj:150]: Good catch. THey're not even supposed to be there - this is supposed to be a copy of the Release PropertyGroup. I'll reorg this a bit to make this kind of thing less likely.

---
In reply to: [520188807](https://github.com/windows-toolkit/Lottie-Windows/pull/380#discussion_r520188807) [](ancestors = 520188807)

File: LottieViewer/LottieViewer.csproj [hunk 2]
```diff
@@ -1,139 +1,96 @@
...
+    <ErrorReport>prompt</ErrorReport>
+    <FileAlignment>512</FileAlignment>
     <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
-    <AppInstallerUpdateFrequency>1</AppInstallerUpdateFrequency>
-    <AppInstallerCheckForUpdateFrequency>OnApplicationRun</AppInstallerCheckForUpdateFrequency>
     <LangVersion>latest</LangVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
     <Nullable>enable</Nullable>
-    <DocumentationFile>bin\LottieViewer.XML</DocumentationFile>
-    <NoWarn>$(NoWarn);2008;8305</NoWarn>
```
Comment (derobinsMSFT) [inline LottieViewer/LottieViewer.csproj:30]: ><NoWarn> [](start = 4, length = 8)

Looks like the `NoWarn` attribute isn't in the most recent changes.

File: LottieViewer/LottieViewer.csproj [hunk 3]
```diff
@@ -1,139 +1,96 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+
```
Comment (derobinsMSFT) [inline LottieViewer/LottieViewer.csproj:4]: Nit: extra spaces.

File: LottieViewer/LottieViewer.csproj (hunk 2 above)
Comment (simeoncran) [reply LottieViewer/LottieViewer.csproj:30]: Yeah, turns out we don't need them any more so I cleaned them out.

---
In reply to: [520810475](https://github.com/windows-toolkit/Lottie-Windows/pull/380#discussion_r520810475) [](ancestors = 520810475)

**Review by derobinsMSFT — DISMISSED** [review]
:shipit:

**Review by derobinsMSFT — APPROVED** [review]
:shipit:

**Pull request merged**